### PR TITLE
Feature/ecom 572 flutter create ci cd pipeline

### PR DESCRIPTION
**Pull request by danial-rokt**
### What Has Changed: ###

CI/CD created
Flutter SDK version 3.0 supported

### Checklist: ###

- [x] I have performed a self-review of my own code.
- [ ] I have made corresponding changes to the documentation.
- [x] I have added tests that prove my fix is effective or that my feature works.
- [x] New and existing unit tests pass locally with my changes.
- [x] I have verified that CI completes successfully.
- [x] All insignificant commits have been squashed.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -1,25 +1,109 @@
...
-  # distribute:
-  #   jobs:
-  #     - distribute
\ No newline at end of file
+          version: 2.10.5
+          filters:
+            branches:
+              only:
+                - /release-(.*)/
+      - test_iOS:
```
Comment (sahil-rokt) [inline .circleci/config.yml:84]: A question, what if the lint job fails? 
Shouldn't test_iOS and test_Android depend on it? This way publish job will also not run.
Same in build_test workflow as well.

Comment (danial-rokt) [reply .circleci/config.yml:84]: Yes but this approach is faster as lint will be triggered concurrently.
If there are any build failures, we cannot merge the PR so it matters if we want to do it quickly or save some CircleCI credit.
I could change it

Comment (sahil-rokt) [reply .circleci/config.yml:84]: yes, I feel better to change, we should not build if lint has failed.

Comment (danial-rokt) [reply .circleci/config.yml:88]: done

**Review by sahil-rokt — APPROVED** [review]

**Pull request merged**